config/v1/types_console.go

```diff
@@ -26,6 +29,26 @@ type Console struct {
 type ConsoleSpec struct {
 	// +optional
 	Authentication ConsoleAuthentication `json:"authentication"`
+	// route contains hostname and serving certificate for replacing the default hostname,
+	// under which is console available.
+	// If not specified, default route will be created by the console-operator.
```

Comment:
Would be good to align on the default behaviour, that means if we wanna keep the default route or not, when the custom hostname will be specified.
Not sure if it makes sense to have two urls for console, if the custom one is defined.

Reply:
I think we do want to keep it, for the squatting reason stated before.
It probably doesn't affect this PR, aside from documentation.

Reply:
Could update to something like:

```
If a custom route is specified, a new route will be created with the provided URL.  The default route will be maintained but will no longer access the console.
```